ee/codegen/src/__test__/__snapshots__/graph-attribute.test.ts.snap

```diff
@@ -1,324 +1,175 @@
+} >> MergeNode
 "
 `;
 
 exports[`Workflow > graph > should be correct for a basic merge node and an additional edge 1`] = `
-"from vellum.workflows import BaseWorkflow
-from .nodes.templating_node import TemplatingNode
-from .nodes.templating_node_2 import TemplatingNode2
-from .nodes.merge_node import MergeNode
-from .nodes.templating_node_3 import TemplatingNode3
-
-
-class TestWorkflow(BaseWorkflow):
-    graph = (
-        {
-            TemplatingNode,
-            TemplatingNode2,
-        }
-        >> MergeNode
-        >> TemplatingNode3
-    )
+"(
+    {
+        TemplatingNode,
+        TemplatingNode2,
+    }
+    >> MergeNode
+    >> TemplatingNode3
+)
 "
 `;
 
 exports[`Workflow > graph > should be correct for a basic merge node case 1`] = `
-"from vellum.workflows import BaseWorkflow
-from .nodes.templating_node import TemplatingNode
-from .nodes.templating_node_2 import TemplatingNode2
-from .nodes.merge_node import MergeNode
-
-
-class TestWorkflow(BaseWorkflow):
-    graph = {
-        TemplatingNode,
-        TemplatingNode2,
-    } >> MergeNode
+"{
+    TemplatingNode,
+    TemplatingNode2,
+} >> MergeNode
 "
 `;
 
 exports[`Workflow > graph > should be correct for a basic merge node case of multiple nodes 1`] = `
-"from vellum.workflows import BaseWorkflow
-from .nodes.templating_node import TemplatingNode
-from .nodes.templating_node_2 import TemplatingNode2
-from .nodes.templating_node_3 import TemplatingNode3
-from .nodes.merge_node import MergeNode
-
-
-class TestWorkflow(BaseWorkflow):
-    graph = {
-        TemplatingNode,
-        TemplatingNode2,
-        TemplatingNode3,
-    } >> MergeNode
+"{
+    TemplatingNode,
+    TemplatingNode2,
+    TemplatingNode3,
+} >> MergeNode
 "
 `;
 
 exports[`Workflow > graph > should be correct for a basic multiple nodes case 1`] = `
-"from vellum.workflows import BaseWorkflow
-from .nodes.templating_node import TemplatingNode
-from .nodes.templating_node_2 import TemplatingNode2
-
-
-class TestWorkflow(BaseWorkflow):
-    graph = {
-        TemplatingNode,
-        TemplatingNode2,
-    }
+"{
+    TemplatingNode,
+    TemplatingNode2,
+}
 "
 `;
 
 exports[`Workflow > graph > should be correct for a basic single edge case 1`] = `
-"from vellum.workflows import BaseWorkflow
-from .nodes.templating_node import TemplatingNode
-from .nodes.templating_node_2 import TemplatingNode2
-
-
-class TestWorkflow(BaseWorkflow):
-    graph = TemplatingNode >> TemplatingNode2
+"TemplatingNode >> TemplatingNode2
 "
 `;
 
 exports[`Workflow > graph > should be correct for a basic single node case 1`] = `
-"from vellum.workflows import BaseWorkflow
-from .inputs import Inputs
-from vellum.workflows.state import BaseState
-from .nodes.search_node import SearchNode
-from .nodes.final_output import FinalOutput
-
-
-class TestWorkflow(BaseWorkflow[Inputs, BaseState]):
-    graph = SearchNode
-
-    class Outputs(BaseWorkflow.Outputs):
-        query = FinalOutput.Outputs.value
+"TemplatingNode
```

Comment:
Switched this from Search -> Templating node to not need to mock document index retrieve in this file's test suite